develop/tutorials/articles/300-osgi-basics/03-adding-third-party-libraries-to-a-module.markdown

```diff
@@ -62,103 +50,94 @@ code like the listing below to your Gradle project:
     apply plugin: "com.liferay.plugin"
 
 If you use Gradle without the `com.liferay.plugin` plugin, you'll have to
-configure your module similar to the way this this tutorial demonstrates for
-Maven and Ivy.  
+[embed the third party libraries wholesale](#embedding-libraries-using-gradle).
 
-To embed a library in your module and make it available in the module's
-classpath automatically, open your module's `build.gradle` file and add the
-library as a dependency in the `compileInclude` configuration:
+$$$
 
-    dependencies {
-        compileInclude group: 'org.apache.shiro', name: 'shiro-core', version: '1.1.0'
-    }
+The recommended package resolution workflow is next. 
 
-The `com.liferay.plugin` plugin's `compileInclude` configuration is transitive.
-It embeds the artifact, and all of the artifact's dependencies in a `lib` folder
-in the module's JAR. Also, it adds the artifact JARs to the `Bundle-ClassPath`
-header in the module's manifest. 
+## Library Package Resolution Workflow [](id=library-package-resolution-workflow)
 
-+$$$
+When you have a dependency on a JAR, much of the time you only need parts of it.
+The OSGi container can resolve dependencies faster if you specify only the
```

Comment:
The resolving dependencies faster really isn't the main goal.  Its making your bundle more modular and also insulating other modules from your depenencies if you are more explicit in what you depend on or include from 3rd party jars.